README.md

```diff
@@ -39,29 +41,35 @@ To ensure that these hooks are installed correctly in your local `.git/hooks` di
 
 This script will copy all the necessary hooks (such as `prepare-commit-msg`) to your local `.git/hooks` directory, ensuring they are executable.
 
+### SwiftLint
+
+We use [SwiftLint](https://github.com/realm/SwiftLint) to enforce Swift style and conventions. Make sure to install it so that linting runs correctly when building.
+
+    brew install swiftlint
```

Comment:
That sounds good, we can think how it might look, though I also don't want to give the bootstrap too many responsibilities, maybe setting up the dev environment (installing stuff) is already too much 🤔